sites/public/src/ListingView.tsx

```diff
@@ -476,8 +476,17 @@ export const ListingView = (props: ListingProps) => {
       <div className="w-full md:w-2/3">
         <header className="image-card--leader">
           <ImageCard
-            imageUrl={imageUrlFromListing(listing, parseInt(process.env.listingPhotoSize))}
+            images={imageUrlFromListing(listing, parseInt(process.env.listingPhotoSize)).map(
+              (imageUrl: string) => {
+                return {
+                  url: imageUrl,
+                }
+              }
+            )}
             tags={getImageCardTag(listing)}
+            moreImagesLabel={t("listings.moreImagesLabel")}
+            moreImagesDescription={t("listings.moreImagesAltDescription") + listing.name}
```

Comment:
For example you could update the listings.moreImagesAltDescription to be `"more images for %{listingName"` and then here you would do `moreImagesDescription={t("listings.moreImagesAltDescription", {listingName: listing.name})}`

Reply:
@ludtkemorgan Oh, I see. I think I'm still confused how that would address the language issue since the listing name would still be at the end of the line. Is there something I'm missing? The only way I could envision this would be with language-specific logic and multiple potential variable positions.

Reply:
Ohhh that makes sense, thank you! @ludtkemorgan